lib/signs/bus.ex

```diff
@@ -434,34 +455,57 @@ defmodule Signs.Bus do
          state
        ) do
     %{top_configs: top_configs, bottom_configs: bottom_configs} = state
+    stop_ids = all_stop_ids(state)
     top_content = configs_content(top_configs, predictions_lookup, route_alerts_lookup)
     bottom_content = configs_content(bottom_configs, predictions_lookup, route_alerts_lookup)
 
     if !Enum.any?(top_content ++ bottom_content, &match?({:predictions, _}, &1)) &&
-         all_stop_ids(state)
+         stop_ids
          |> Enum.all?(fn stop_id ->
            Map.get(stop_alerts_lookup, stop_id) in @alert_levels
          end) do
       no_service_content()
     else
+      silver_line_headway =
+        combine_silver_line_headway_messages(
```

Comment:
Taken from how we handle rail. 

If the ranges are the same, we combine them into one headway message, otherwise we don't use it.